x/mongo/driver/mongocrypt/mongocrypt.go

```diff
@@ -244,6 +255,22 @@ func (m *MongoCrypt) CreateExplicitDecryptionContext(doc bsoncore.Document) (*Co
 	return ctx, nil
 }
 
+// CSFLEVersion returns the version number for the loaded csfle library, or 0 if the csfle library
+// was not loaded.
+func (m *MongoCrypt) CSFLEVersion() uint64 {
+	return uint64(C.mongocrypt_csfle_version(m.wrapped))
+}
+
+// CSFLEVersionString returns the version string  for the loaded csfle library, or an empty string
```

Comment:
```suggestion
// CSFLEVersionString returns the version string for the loaded csfle library, or an empty string
```

Reply:
The "csfle" -> "crypt_shared" rename broke this suggestion, but I've fixed this typo in the comment.